.github/workflows/csv-coverage.yml

```diff
@@ -38,7 +22,7 @@ jobs:
       uses: actions/checkout@v2
       with:
         path: codeqlModels
-        ref: github.event.inputs.qlModelShaOverride
+        ref: ${{ github.event.inputs.qlModelShaOverride }}
```

Comment:
You can combine these two checkout steps for `codeqlModels`: use `ref: '${{ github.event.inputs.qlModelShaOverride || github.ref }}` and it will fall back to the dispatch ref if the input is missing.